android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -170,9 +171,19 @@ public Completable resendEmailVerification(@NonNull final String email) {
                     accountDAO.setStudyParticipant(
                             new StudyParticipant()
                                     .email(signIn.getEmail()));
+                }).doOnError(throwable -> {
+                    // a 412 is a successful signin
+                    if (throwable instanceof ConsentRequiredException) {
```

Comment:
When I was writing junit tests and signing in, I noticed that while
unconsented,  the signIn, email, etc weren't being cached in AuthManager
like we do for a successful sign in